js/tests/unit/tooltip.js

```diff
@@ -389,17 +392,24 @@ $(function () {
     $topTooltip.bootstrapTooltip('hide')
     equal($('.tooltip').length, 0, 'top positioned tooltip removed from dom')
 
-    var $rightTooltip = $('<div style="right: 0;" rel="tooltip" title="Right tooltip">Right Dynamic Tooltip</div>')
+    var $rightTooltip = $('<a rel="tooltip" title="Right tooltip">Right Dynamic Tooltip</a>')
       .appendTo($container)
       .bootstrapTooltip({ placement: 'right auto' })
 
     $rightTooltip.bootstrapTooltip('show')
+    ok($('.tooltip').is('.right'), 'left positioned tooltip is positioned right')
```

Comment:
The terminology here is a bit confusing, but when these tests refer to an "X positioned tooltip", they're referring to the value of the `placement` argument. So the reference to a "left positioned tooltip" here is wrong. Same with some of the other tests you added.